docs_src/microservices/configuration/CommonEnvironmentVariables.md

```diff
@@ -161,24 +161,46 @@ Any configuration setting from a service's `configuration.toml` file can be over
 <TOML-SECTION-NAME>_<TOML-SUB-SECTION-NAME>_<TOML-KEY-NAME>
 ```
 
-!!! edgey "EdgeX 2.0"
-    With EdgeX 2.0 the use of CamelCase environment variable names is no longer supported. Instead the variable names must be all uppercase as in the example below. Also the using of dash `-` in the TOML-NAME is converted to an underscore `_` in the environment variable name.
-
-!!! example "Example - Environment Overrides of Configuration"
+!!! example "Example - Environment Overrides of Configuration core/support/device/app/security services"
 
 ~~~toml
```

Comment:
remove this

Reply:
Humm, not sure how those got in there... Removed